examples/tls/config.yaml

```diff
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=../../schema/local.json
+# yaml-language-server: $schema=https://raw.githubusercontent.com/agentgateway/agentgateway/refs/heads/main/schema/local.json
```

Comment:
Hmm, this is good in some ways but not in others. Since if you are checking out the repo, you will not want to check `main` but the current file...